smartmeter/display.cpp

```diff
@@ -22,36 +22,22 @@ std::vector<String> Display::strings = {
 };
 
 Display::Display() :
+    
```

Comment:
Do not edit lines that you don't need to edit, this can reflect in conflicts in future ! Thing about that.

Reply:
You're right